frontend/packages/knative-plugin/src/components/overview/FilterTable.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+
+import cx from 'classnames';
+import { TableVariant, Table, TableBody } from '@patternfly/react-table';
+import './FilterTable.scss';
+
+export type FilterTableRowProps = { key: string; value: string }[];
+type FilterTableProps = {
+  filters: FilterTableRowProps;
+  bordered?: boolean;
+  paddingLeft?: boolean;
+};
+
+const FilterTable: React.FC<FilterTableProps> = ({
+  filters,
+  bordered = true,
+  paddingLeft = false,
+}) => {
+  const filterRow = (key: string, value: string) => {
+    const className = cx({ 'kn-filter-table-row-bordered': bordered });
+    return {
+      cells: [
+        {
+          title: key,
+          props: {
+            className: cx(className, { 'kn-filter-table-padding-left': paddingLeft }),
```

Comment:
```suggestion
            className: cx(className, { 'kn-filter-table--padding-left': paddingLeft }),
```

Reply:
Will update to `kn-filter-table__padding--left`